Jenkinsfile_parameterized

```diff
@@ -2,6 +2,27 @@
 
 @Library("Infrastructure") _
 
+List<LinkedHashMap<String, Object>> secrets = [
+  secret('s2s-url-for-tests', 'S2S_URL_FOR_TESTS'),
+  secret('s2s-name-for-tests', 'S2S_NAME_FOR_TESTS'),
+  secret('s2s-secret-for-tests', 'S2S_SECRET_FOR_TESTS'),
+  secret('idam-url-for-tests', 'IDAM_URL_FOR_TESTS'),
+  secret('idam-username-for-tests', 'IDAM_USERNAME_FOR_TESTS'),
+  secret('idam-password-for-tests', 'IDAM_PASSWORD_FOR_TESTS'),
+  secret('idam-client-id-for-tests', 'IDAM_CLIENT_ID_FOR_TESTS'),
+  secret('idam-client-secret-for-tests', 'IDAM_CLIENT_SECRET_FOR_TESTS'),
+  secret('idam-redirect-uri-for-tests', 'IDAM_REDIRECT_URI')
```

Comment:
Probably better for you to output this from the terraform its not a secret?

Reply:
any outputs from terraform will be automatically converted to env variables,
but don't do this for secrets

Reply:
nice suggestion

Reply:
I didn't know about that. That's what I'm going to do, then.